### ISSUE TITLE
"require-dependent-keys" false negative with computed decorator

### ISSUE THREAD

**Issue by allthesignals** [ember-best-practices/eslint-plugin-ember-best-practices#97]
```
import Model from 'ember-data/model';
import { attr, belongsTo } from 'ember-decorators/data';
import { computed } from 'ember-decorators/object';

export default class LayerModel extends Model {
  @belongsTo('layer-group') layerGroup
  @attr() style

  @computed('layerGroup.visible') visible
}
```

Throws:
> Do not use Computed Properties without dependent keys...

![image](https://user-images.githubusercontent.com/5004319/38461351-72537d70-3a9b-11e8-9240-5e9e5ccb4793.png)

package.json:
```
"ember-source": "~3.0.0",
//...
"eslint-plugin-ember-best-practices": "1.1.1",
"eslint-plugin-ember": "^5.0.0",
```

**Comment by allthesignals** [comment]
Maybe it's just trying to get me to use an alias there instead ;)

**Comment by allthesignals** [comment]
Added a [failing test](https://github.com/allthesignals/eslint-plugin-ember-best-practices/blob/c94b38955701cce0ff4beaa2efffe2fd5d69f63d/tests/lib/rules/require-dependent-keys.js#L21-L29) but not quite sure how to get the decorator modules imported and parsing correctly with class syntax...
